kuma/api/v1/views.py

```diff
@@ -108,6 +108,10 @@ def document_api_data(doc=None, ensure_contributors=False, redirect_url=None):
     else:
         en_slug = ''
 
+    other_translations = doc.get_other_translations(
+        fields=('locale', 'slug', 'title'))
+    all_locales = set([doc.locale]) | set(t.locale for t in other_translations)
```

Comment:
nit: I had a moment where I thought you were using get_hreflang wrong and actually passing all locales to it. This would be slightly clearer if the variable name was translation_locales or available_locales or something like that.

Reply:
Good point, I like `available_locales` better, thanks!